src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx

```diff
@@ -6,12 +6,14 @@ import { getMostVulnerableSeatByQuotient } from "../../../utilities/district";
 import { norwegian } from "../../../utilities/rt";
 import { isQuotientAlgorithm } from "../../../computation/logic";
 import { VulnerableDistrictSeatText } from "./VulnerableDistrictSeatText";
+import { Dictionary } from "lodash";
 
 export interface DistrictOverviewProps {
     districtResults: DistrictResult[];
     districtWidth: number;
     decimals: number;
     algorithm: AlgorithmType;
+    partyMap: Dictionary<string>;
```

Comment:
We are using a lodash type for this? Is this what you intended? I think we have a utility somewhere that should be removed if we're migrating.